### PR TITLE
fix(ins): update flex ip description

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: compute/instances/how-to/use-flexips.mdx [hunk 1]
```diff
@@ -5,33 +5,42 @@ meta:
...
 
-This page shows you how to use [flexible IP addresses](/compute/instances/concepts/#flexible-ip). When you create an Instance, it gets the following by default:
-  - A [public IP address](/compute/instances/concepts/#public-ip), which is also a flexible IP address
-  - A [private IP address](/network/public-gateways/concepts/#private-ip-address)
+This page provides guidance on using [flexible IP addresses](/compute/Instances/concepts/#flexible-ip) in your Scaleway Instances. 
 
-You can hold flexible IP addresses independently of any Instance, and attach and detach them to/from any of your Instances. You can keep a number of flexible IP addresses in your account at any given time. Flexible IP addresses allow you to do [live migration of IP addresses between your Instances](/compute/instances/how-to/migrate-instances/).
+When creating an Instance, you have options for network connectivity:
+
+- **Flexible Routed IP addresses**: These are public IP addresses routed directly to your Instance.
```
Comment (jcirinosclwy) [inline compute/instances/how-to/use-flexips.mdx:20]:
```suggestion
- **Flexible Routed IP addresses**: public IP addresses routed directly to your Instance.
```

File: compute/instances/how-to/use-flexips.mdx [hunk 2]
```diff
@@ -5,33 +5,42 @@ meta:
...
+- An [Instance](/compute/Instances/how-to/create-an-Instance)
 
 <Macro id="flexible-ip" />
 
 ## How to detach a flexible IP address from an Instance
 
 Detaching your Instance's flexible (public) IP address cuts any ongoing public connection to the Instance. This feature gives you a lot of control if, for example, you want to put your Instances into a Private Network so that they can communicate between themselves without being exposed to the internet. This is particularly useful for Instances hosting databases, backups, or sensitive content. It also allows you to optimize your costs and fight the global IPv4 shortage.
 
-1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
+1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/Instance/servers) displays.
```
Comment (jcirinosclwy) [inline compute/instances/how-to/use-flexips.mdx:43]: We don't need to have uppercase on links, do we?

**Review by jcirinosclwy — APPROVED** [review]

File: compute/instances/how-to/use-flexips.mdx [hunk 3]
```diff
@@ -5,33 +5,42 @@ meta:
...
+<Message type="tip">
+  Instances using routed public IPv4 addresses can also use flexible IPv6 addresses.
+</Message>
 
 <Macro id="requirements" />
 
 - A Scaleway account logged into the [console](https://console.scaleway.com)
 - [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
-- An [Instance](/compute/instances/how-to/create-an-instance)
+- An [Instance](/compute/Instances/how-to/create-an-Instance)
```
Comment (bene2k1) [inline compute/instances/how-to/use-flexips.mdx:35]:
```suggestion
- An [Instance](/compute/instances/how-to/create-an-instance)
```

File: compute/instances/how-to/use-flexips.mdx (hunk 2 above)
Comment (bene2k1) [inline compute/instances/how-to/use-flexips.mdx:43]:
```suggestion
1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
```

File: compute/instances/how-to/use-flexips.mdx [hunk 4]
```diff
@@ -54,7 +63,7 @@ Detaching your Instance's flexible (public) IP address cuts any ongoing public c
 
 ## How to attach an existing flexible IP address to an Instance
 
-1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
+1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/Instance/servers) displays.
```
Comment (bene2k1) [inline compute/instances/how-to/use-flexips.mdx:66]:
```suggestion
1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
```

File: compute/instances/how-to/use-flexips.mdx [hunk 5]
```diff
@@ -66,7 +75,7 @@ Detaching your Instance's flexible (public) IP address cuts any ongoing public c
 
 When you no longer want one of your existing flexible IP addresses, you can delete it from your account, which releases it to be used by other users.
 
-1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
+1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/Instance/servers) displays.
```
Comment (bene2k1) [inline compute/instances/how-to/use-flexips.mdx:78]:
```suggestion
1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
```

**Review by ofranc — APPROVED** [review]

**Pull request merged**